clickhouse/templates/statefulset-clickhouse-replica.yaml

```diff
@@ -116,9 +116,11 @@ spec:
           mountPath: /etc/clickhouse-server/metrica.d
         - name: {{ include "clickhouse.fullname" . }}-users
           mountPath: /etc/clickhouse-server/users.d
+        {{- if .Values.clickhouse.securityContext.enabled }}
```

Comment:
continuation:

assuming you removed the `enabled` property you can make this conditional like so:

```yaml
{{- with .Values.clickhouse.securityContext }}
        securityContext:
          {{- toYaml . | nindent 10 }}
{{- end }}
``` 

that way the `securityContext` will only be included if the `.Values.clickhouse.securityContext` is not nil or empty

this way the content of the `securityContext` is also flexible because it can be set from the `values.yaml` without requiring a property for every possible field to be set